### ISSUE TITLE
Error "insertClipping is not defined" when attempting to insert clipping if extension installed while compose window already open

### ISSUE THREAD

**Issue by aecreations** [aecreations/clippings-tb#158]
Clippings for Thunderbird 7.0a0+ on Thunderbird 115

Steps to repro:
1. Open a new compose window
2. Install Clippings for Thunderbird
3. Create a clipping
4. Right-click on the compose window from step 1 and select the newly-created clipping

Result: The clipping isn't pasted. The error console shows the following message:
```
Uncaught (in promise) Error: insertClipping is not defined
apply                  self-hosted:2366
applySafeWithoutClone  resource://gre/modules/ExtensionCommon.sys.mjs:635
wrapPromise            resource://gre/modules/ExtensionCommon.sys.mjs:963
```
No source file or line number is provided.

This doesn't occur with new compose windows that are opened after the extension is installed.

**Comment by aecreations** [comment]
Question posted to the Thunderbird developers forum: https://thunderbird.topicbox.com/groups/addons/T8b4c0c6af5b4ffac/compose-script-not-loading-in-compose-windows-already-opened-when-extension-installed

**Comment by aecreations** [comment]
Fixed in 7.0 alpha 1
